spec/schema.json

```diff
@@ -16,53 +16,346 @@
+        "tags": [
+          "Strike Off Objections"
+        ],
+        "responses": {
+          "200": {
+            "description": "the attachment was found",
+            "schema": {
+              "type": "file"
+            }
+          },
+          "404": {
+            "description": "Objection or attachment not found"
           }
         }
       }
     }
   },
   "definitions": {
-    "ObjectionOutput": {
+    "Objection": {
       "type": "object",
       "properties": {
-        "requestId": {
-          "type": "integer",
-          "format": "int64"
+        "id": {
+          "type": "string",
+          "format": "uuid"
+        },
+        "created_on": {
+          "type": "string",
+          "format": "date-time"
+        },
+        "created_by": {
+          "$ref": "#/definitions/CreatedBy"
+        },
+        "reason": {
+          "type": "string"
+        },
+        "attachments": {
+          "type": "array",
+          "items": {
+            "properties": {
+              "link": {
+                "type": "string",
+                "format": "uri"
+              }
+            }
+          }
+        },
+        "status": {
+          "$ref": "#/definitions/Status"
+        }
+      }
+    },
+    "StrikeOffStatus": {
+      "type": "object",
+      "properties": {
+        "isEligible": {
+          "type": "boolean"
+        },
+        "strikeOffDate": {
+          "type": "string"
         }
       }
     },
-    "ObjectionInput": {
+    "ResourceCreated": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "format": "uuid"
+        }
+      }
+    },
+    "Attachment": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "format": "uuid"
+        },
+        "name": {
+          "type": "string"
+        },
+        "content_type": {
+          "type": "string"
+        },
+        "size": {
+          "type": "number"
+        },
+        "links": {
+          "type": "object",
+          "properties": {
+            "download": {
+              "type": "string",
+              "format": "uri"
+            },
+            "self": {
+              "type": "string",
+              "format": "uri"
+            }
+          }
+        }
+      }
+    },
+    "CreatedBy": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string"
+        },
+        "forename": {
+          "type": "string"
+        },
+        "surname": {
+          "type": "string"
+        },
+        "email": {
+          "type": "string",
+          "format": "email"
+        }
+      }
+    },
+    "PatchRequest": {
       "type": "object",
       "properties": {
         "reason": {
           "type": "string"
+        },
+        "status": {
+          "$ref": "#/definitions/Status"
         }
       }
+    },
+    "Status": {
+      "type": "string",
+      "enum": [
+        "open",
```

Comment:
Personally I like enum values in uppercase. I just think it helps them stand out (a bit like a constant) but up to you.

Reply:
updated